src/hypergraph.jl

```diff
@@ -295,4 +295,39 @@ function nhv(h::Hypergraph{T, V, E}) where {T <: Real, V, E}
     length(h.v2he)
 end
 
+"""
+    _walk!(h::Hypergraph, s::Vector{Int}, i::Vector{Int}, visited::Vector{Bool})
```

Comment:
see comment below + it is OK to leave the docstring (although we do not expose it in the docs, as it might be useful for the readers)
```suggestion
    _walk!(h::Hypergraph, s::AbstractVector{Int}, i::Int, visited::AbstractVector{Bool})
```